__tests__/app.test.js

```diff
@@ -367,6 +367,61 @@ describe('DELETE /api/comments/:comment_id', () => {
       })
     
   })
-  
+})
+describe('GET /api/users', () => {
+  it('GET: 200 - Should respond with an array of user objects', () => {
+    return request(app)
+    .get('/api/users')
+    .expect(200)
+    .then(({ body }) => {
+        expect(Array.isArray(body.users)).toBe(true);
+        expect(body.users.length).toBeGreaterThan(0);
+        body.users.forEach((user) => {
+          expect(user).toHaveProperty('username');
+          expect(user).toHaveProperty('name');
+          expect(user).toHaveProperty('avatar_url');
+          expect(typeof user.username).toBe('string');
```

Comment:
```js
expect(user).toHaveProperty('avatar_url');
expect(typeof user.username).toBe('string');
```
this can be achieved in one line with
```js
expect(user).toHaveProperty('avatar_url', expect.any(String));
```